contracts/PCSV3PositionUtils.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IPCSV3NonfungiblePositionManager as IPCSV3NPM, INonfungiblePositionManager as INPM} from "@aperture_finance/uni-v3-lib/src/interfaces/INonfungiblePositionManager.sol";
+import {NPMCaller, PositionFull} from "@aperture_finance/uni-v3-lib/src/NPMCaller.sol";
+import {PoolAddressPancakeSwapV3} from "@aperture_finance/uni-v3-lib/src/PoolAddressPancakeSwapV3.sol";
+import {IUniswapV3PoolState, V3PoolCallee} from "@aperture_finance/uni-v3-lib/src/PoolCaller.sol";
+import {ERC20Callee} from "./libraries/ERC20Caller.sol";
+import {PoolUtils} from "./PoolUtils.sol";
+
+struct PCSV3Slot0 {
+    uint160 sqrtPriceX96;
+    int24 tick;
+    uint16 observationIndex;
+    uint16 observationCardinality;
+    uint16 observationCardinalityNext;
+    uint32 feeProtocol;
```

Comment:
Can change the original `Slot0. feeProtocol` to `uint32` and reduce a lot of duplicates.